jack.py

```diff
@@ -674,6 +675,38 @@ def transport_stop(self):
         """Stop JACK transport."""
         _lib.jack_transport_stop(self._ptr)
 
+    def set_timebase_callback(self,
+                              conditional, timebase_callback, userdata=None):
+        """Register as timebase master for the JACK subsystem.
+
+        The timebase master registers a callback that updates extended position
```

Comment:
Please limit the line length of docstrings to 72 characters, as of [PEP8](http://legacy.python.org/dev/peps/pep-0008/#maximum-line-length).